### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/layout_moyo-header-en/)
[TEST REPORT LINK](https://lefteris-l.github.io/layout_moyo-header-en/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,55 @@
...
+          alt="logo"
+          class="logo_image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
```
Comment (mgrinko) [inline src/index.html:28]: I suggest you putting 1 blank line between sibling tags (when they have some content) to improve readability

File: src/style.css [hunk 2]
```diff
@@ -1 +1,60 @@
...
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.nav_link {
+  position: relative;
+  height: 60px;
+  white-space: nowrap;
+  display: flex;
+  align-items: center;
```
Comment (mgrinko) [inline src/style.css:37]: I'd put flex rules at the top to make them more noticeable

File: src/style.css [hunk 3]
```diff
@@ -1 +1,60 @@
...
+}
+
+.nav_link.is-active,
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  border-radius: 5px;
```
Comment (mgrinko) [inline src/style.css:53]: this rule is just a styling so may go later 
to have the most important rules at the top

**Review by mgrinko — APPROVED** [review]

File: src/style.css (hunk 3 above)
Comment (lefteris-L) [reply src/style.css:53]: Noted. Thank you Misha

**Pull request closed without merging**